theatre/studio/src/store/types/ahistoric.ts

```diff
@@ -1,8 +1,13 @@
 import type {ProjectState} from '@theatre/core/projects/store/storeTypes'
 import type {IRange, StrictRecord} from '@theatre/shared/utils/types'
+import type {
+  Keyframe,
+  ISelectedKeyframes,
+} from '@theatre/core/projects/store/types/SheetState_Historic'
 
 export type StudioAhistoricState = {
   visibilityState: 'everythingIsHidden' | 'everythingIsVisible'
+  keyframesClipboard: ISelectedKeyframes
```

Comment:
What happens when the user upgrades from `0.4.7` to the new version?

Reply:
We're using the [getCopiedKeyframes](https://github.com/theatre-js/theatre/pull/65/files#diff-ba3a4c0d5523e31b3109feea4e46de5e6a7da4afaad01f3bc4a1d752d42892f2R91) selector which provides a fallback. Do you think this is sufficient, or have any suggestions to make this more robust / future proof?

Reply:
Maybe this ok for now, but we can implement something more robust later